proto/src/main/proto/pb.proto

```diff
@@ -1709,7 +1709,6 @@ message Trade {
 
     Offer offer = 1;
     ProcessModel process_model = 2;
-    string taker_fee_tx_id = 3 [deprecated = true];
```

Comment:
The numbering should be reset too, please.

Reply:
Done.